### ISSUE TITLE
Initial styleswitcher theme looks off

### ISSUE THREAD

**Issue by wasamasa** [mobius3/KiWi#22]
![](http://i.imgur.com/C5ugWXo.png)

Note the discontinuous line and rectangle rendering.  This does curiously not happen with the other themes.

**Comment by mobius3** [comment]
Yeah, that happens because of the tile stretch hint enabled by default in frames. This is fixed by setting the hint to false when using this tileset.

A more elaborate solution would be analysing the pixels in the filling tiles and detect automatically if they can be stretched. I'll do that later. Thanks for your report!
